services/brig/src/Brig/User/Template.hs

```diff
@@ -217,6 +218,13 @@ loadUserTemplates o = readLocalesDir defLocale templateDir "user" $ \fp ->
             <*> pure emailSender
             <*> readText fp "email/sender.txt"
         )
+    <*> ( VerificationEmailTemplate activationUrl
+            <$> readTemplate fp "email/verification-subject.txt"
+            <*> readTemplate fp "email/verification-login.txt"
+            <*> readTemplate fp "email/verification.html"
```

Comment:
Isn't the wording a bit different in the txt mail from the html mail? Or what was the reason for introducing a new txt mail but not a corresponding html mail?

Reply:
Yes (we found that independently in a phone call).